src/main/webapp/signin.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>FeedOn's Sign In Page</title>
+    <script src="js/client.js" defer></script>
+</head>
+
+<body onload="getSignIn()">
+</body>
+
+</html>
```

Comment:
Little note of the red (-) error at the end. We need an extra blank new line at the end of all these files. 

(To my understanding, the empty line in the end of file appears so that standard reading from the input stream will know when to terminate the read, usually returns EOF to indicate that you have reached the end.)

Reply:
alright, I will make this edit